### PR TITLE
Contact Form Validation & Styling

### PR DESCRIPTION
**Pull request by GeekyHacks**
- Apply all inquired validations for all inputs.
- Style the form container according to the exercise requirements.
- Style the form elements according to the exercise requirements.
- Finalize the styling for the contact me section.

### CODE REVIEW

File: mobilestyle.css [hunk 1]
```diff
@@ -629,39 +639,72 @@ button:active {
...
+  display: grid;
+  grid-template-columns: repeat(8, 1fr);
+  gap: 10px;
+  grid-auto-rows: 100px;
+  grid-template-areas:
+    "a a a a b b b b"
+    "a a a a b b b b"
+    "c c c c d d d d"
+    "c c c c d d d d";
+} */
```
Comment (Ibizugbe) [inline mobilestyle.css:695]: Kindly ensure to take away every commented piece of code as they make your codebase look a bit messy😉.

File: index.html [hunk 2]
```diff
@@ -466,7 +467,7 @@ <h2 class="contact-me-header-desktop">
             name="message"
           ></textarea>
           <button
-          id="submitBtn"
+            id="submitBtn"
```
Comment (Ibizugbe) [inline index.html:470]: Your form section looks a bit different from the given design😉. Kindly consider the spacing around the highlighted text in the image below and make the changes where necessary👍.

yours
image

design
image

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Team👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Linters are correctly setup and they are passing👍.
- [x] Github flow have been used👍.
- [x] All input fields have correct type attributes and they are wrapped in a single form element👍.
- [x] Form have been integrated to make use of the `formspre` service👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by NahnahAJ — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @GeekyHacks , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- All requested changes implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**